svgdigitizer/svgplot.py

```diff
@@ -36,15 +80,15 @@ def __init__(self, filename, xlabel=None, ylabel=None, sampling_interval=None):
 
         self.get_parsed()
         self.create_df()
-    
+
     @cached_property
     def transformed_sampling_interval(self):
         factor = 1/((self.trafo['x'](self.sampling_interval)-self.trafo['x'](0))/(self.sampling_interval/1000))
         return factor
-    
+
     def get_points(self):
         '''Creates:
-        ref_points: relative values of the spheres/eclipses in the svg file.
+        ref_points: relative values of the spheres/ellipses in the svg file.
```

Comment:
Actually which spheres/ellipses. I think this is a relic from the first modified version, where spheres were located on the axis.